cli/index.ts

```diff
@@ -366,11 +369,13 @@ const builtYargs = createYargsCli({
             watching = false;
             process.exit(1);
           });
+          if (!watching) {
```

Comment:
i don't *believe* this line can ever be called? (or if it is, it gets called when the watcher is about to start running, but this will have the effect of killing the process at that time)

i'd expect this to be above the `if (argv.watch) {` line, and only return if `!argv.watch`

Reply:
👍